src/actions/helpers.js

```diff
@@ -21,22 +29,19 @@ export function requestFailure(type, error) {
     };
 }
 
-export function emptyError() {
-    return {
-        id: '',
-        message: '',
-        detailed_error: '',
-        status_code: ''
-    };
-}
-
 export function bindClientFunc(clientFunc, request, success, failure, ...args) {
     return async (dispatch, getState) => {
         dispatch(requestData(request), getState);
 
         try {
             const data = await clientFunc(...args);
-            dispatch(requestSuccess(success, data), getState);
+            if (Array.isArray(success)) {
```

Comment:
Cool, I like how this looks. We could even consider batching them in the future if necessary